sda/host.go

```diff
@@ -612,3 +621,74 @@ func (h *Host) Rx() uint64 {
 func (h *Host) Address() string {
 	return h.workingAddress
 }
+
+//Received gives the total number of packets received by a host
+func Received(n map[network.ServerIdentityID]network.SecureConn) uint64 {
+	var a uint64
+	for _, value := range n {
+		a = value.Rx()
+	}
+	return a
+}
+
+//Sent gives the total number of packets sent by the host
+func Sent(n map[network.ServerIdentityID]network.SecureConn) uint64 {
+	var a uint64
+	for _, value := range n {
+		a = value.Tx()
+	}
+	return a
+
+}
+
+//ID is the host that you are asking for stats from
+func ID(n map[network.ServerIdentityID]network.SecureConn) string {
+	var a string
+	for _, value := range n {
+		a = value.Local()
+	}
+	return a
+
+}
+
+//Remote is who the host is connected to
+func Remote(n map[network.ServerIdentityID]network.SecureConn) string {
+	var a string
+	for _, value := range n {
+		a = a + value.Remote() + "\n"
+	}
+	return a
+}
+
+//GetStatus is a function
+func (h *Host) GetStatus() Status {
+	m := make(map[string]string)
```

Comment:
I wonder whether we should allow spaces in the key-name of that map. It might give us problems later on if we want to work on those key-names. I propose we use "_" instead of spaces.